bots/vpc_isolate.py

```diff
@@ -0,0 +1,174 @@
+'''
+##vpc-isolate
+What it does: turn off dns resource
+              change network acl to new empty one with deny all
+              add iam policy, to all users in the account, which limits vpc use: ec2 and sg use in the vpc
+
+Usage: AUTO: vpc_isolate
+Limitation: None
+'''
+
+import boto3
+from botocore.exceptions import ClientError
+import json
+
+
+def create_acl(vpc):
+
+    text_output = ''

```

Comment:
you can define text_output as global so you will be able to modify it without re-declare it